adala/runtimes/_litellm.py

```diff
@@ -131,6 +132,7 @@ def record_to_record(
         extra_fields: Optional[Dict[str, str]] = None,
         field_schema: Optional[Dict] = None,
         instructions_first: bool = False,
+        response_model: Optional[Type[BaseModel]] = None,
```

Comment:
can you raise a NotImplementedError in this function if `response_model` is provided, since nothing is done with it yet?

Reply:
oh, it lost in merging, fixing implementation now